x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/mapper/ShapeFieldMapperTests.java

```diff
@@ -72,6 +72,12 @@ protected void registerParameters(ParameterChecker checker) throws IOException {
         });
     }
 
+    @Override
+    protected boolean supportsSearchLookup() {
+        // TODO: is this really true? We have failing tests, but they look like test issues not core issues
+        return false;
```

Comment:
It seems that this was not needed before because we did not have doc values so it was equivalent to set this to false. Now we need to do it explicitly.